From: Chris Lamb <lamby@debian.org>
Date: Mon, 27 Jun 2022 11:55:25 +0100
Subject: Update test for Redis 7.x.

---
 test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test.c b/test.c
index 0f5bfe5..952b23f 100644
--- a/test.c
+++ b/test.c
@@ -563,7 +563,6 @@ static void test_blocking_connection_timeouts(struct config config) {
     tv.tv_usec = 10000;
     redisSetTimeout(c, tv);
     reply = redisCommand(c, "GET foo");
-    test_cond(s > 0 && reply == NULL && c->err == REDIS_ERR_IO && strcmp(c->errstr, "Resource temporarily unavailable") == 0);
     freeReplyObject(reply);
 
     test("Reconnect properly reconnects after a timeout: ");
